optax/_src/loss.py

```diff
@@ -503,3 +503,19 @@ def kl_divergence_with_log_targets(log_predictions: chex.Array,
   chex.assert_type([log_predictions, log_targets], float)
   loss = jnp.exp(log_targets) * (log_targets - log_predictions)
   return jnp.sum(loss, axis=-1)
+
+def hinge_loss(predictions: chex.Array, targets: chex.Array) -> chex.Array:
+  """
+  Computes the hinge loss for Binary Classification.
+
+  Args:
+    predictions: Predicted values with shape [..., dim].
```

Comment:
Could you add more explanation that predictions aren't the class predictions but the output of the decision function e.g. w*x + b. Perhaps we can also find a name that reflects this? I'm not sure I like the sklearn name `pred_decision` though. Perhaps `predictor_outputs`?